### ISSUE TITLE
documentation / wiki clarification on the split keyboard status

### ISSUE THREAD

**Issue by alexmaloteaux** [customMK/Bonsai-C#2]
HI,

after ordering the part for a bonsai-c , mainly to deal with the 32k limitation of the pro-micro, i realized  that the proton-c is actually not yet compatible with split keyboard like corne, lotus58 ... when using qmk

Is it possible to clarify the actual status of bonsai-c regarding split keyboard witj qmk and have it eventually in the readme ?

I guess that ` a drop-in-replacement for Proton C rev 2` is enough for most people, but im pretty new to qmk and was not  aware of that limitation. 

Best Regards

**Comment by infinityis** [comment]
Yes, I can add some clarifications to the readme in that regard. Unfortunately it seems like almost any STM32F-based replacement may have some sort of tradeoffs with regards to compatibility.

I do want to point out that (based on my current understanding) you can achieve some compatibility following this guide from Drashna for Kyria (and other split keyboards), which gets it working by leave out one header pin and adding add two wires (one for split comms, one for RGB) https://github.com/qmk/qmk_firmware/blob/develop/keyboards/splitkb/kyria/readme.md#proton-c-conversion-guide

Something else that should be do-able is to create a separate version of Bonsai C; instead of targeting compatibility with Proton C it is can be made specifically as a drop-in replacement for split keyboards. That, or (like I'm doing on the Bonsai C4 design now) adding a few solder bridges; this could make it very easy to switch between Proton C compatibility and split keyboard compatibility.

Thoughts? I don't have a split keyboard, so if the incompatibility extends any further than the modifications I linked to, let me know or send links to more information so I can try to accommodate.

**Comment by alexmaloteaux** [comment]
ok, once i get mine soldered, i will review and test on a lotus58 first and report here. 
There is another wiring example in this project :

https://github.com/ItsWaffIe/waffle_corne.

The misleading documentation comes actually from qmk  wiki where they claim the support is partial, upcoming...

https://beta.docs.qmk.fm/using-qmk/hardware-features/proton_c_conversion
https://qmk.fm/proton-c/
